chapter1_welcome/chapter1_welcome.md

```diff
@@ -0,0 +1,153 @@
+The Components folder contains examples of individual self-contained operations that can be performed using the API. These examples can be used to learn how basic semantic steps are implemented and what associated parameters and return values are involved. Component examples include:
+-	Create a report.
+-	Upload a file.
+-	Add an operator.
+Each component example is implemented in a function that can be included in other code. In this way, components can be grouped together to implement larger workflows.
+
+### Workflows
+The Workflows folder contains examples of how more elaborate functionality is implemented by aggregating various component examples together. These examples can be used to learn how the microservices and technologies work together in a broader context. Workflow examples include:
+-	Convert a design file into a task able to be viewed on a machine control client.
+-	Stream live information about the machines connected to a site using SmartView (chapter 7).
+-	Configure a site with the metadata required to run the Topcon Haul App.
+## Prerequisites
+There are a few steps that need ticking off before we can start using the API. These prerequisites are listed below and covered in our Topcon Software Developer Network (TSDN) page at https://developer.topcon.com. 
+### Create a Topcon Account
+Creating a Topcon account is free. Follow the instructions at https://developer.topcon.com/en/gwWiCQ. Once you have an account, you can be invited to an Organization or create your own. 
+### Create or Join an Organization
+Creating an Organization is free. Follow the instructions at https://developer.topcon.com/en/gwWiCQ. Existing Organization owners can invite you to their organizations via email.
+### Service Points
+The Sitelink3D v2 API is charged under a consumption-based model. Consumption is reconciled by the deduction of preloaded service points. Service points are associated with an Organization and are consumed when data is logged to a site via our data-logger microservice on a per-client, per-day basis. Operations that do not write to data-logger will not consume service points. Connecting a client such as the Haul Truck app or 3DMC to a Sitelink3D v2 site will consume service points.
+Although service points are required on both our QA and production environments, they are priced differently as described in the “Cloud Environments” section below. 
+### Obtain API Credentials
+Credentials are comprised of the following:
+-	Client ID.
+-	Client Secret.
+-	Access Token.
+-	Service Points for the organization (production only)
+
+Obtaining these is explained at https://developer.topcon.com/en/igWiCQ. Note that step 5 of Method 2 is demonstrated in this example repository under components/tokens/get_token.py  
+## Cloud Environments
+Sitelink3D v2 is available at two isolated environments:
+-	QA.
+-	Production.
+
+Our QA environment is a sand box intended for early testing of features before being deployed to production. 
```

Comment:
Give QA and Production their own subheadings and allocate the information underneath them